src/App.js

```diff
@@ -1,18 +1,25 @@
 import { Component } from 'react';
-import logo from './react-logo.svg';
 import './App.css';
 import React from 'react';
+import Header from './Header';
+import CreatureList from './CreatureList';
+import creatures from './creatures-data';
```

Comment:
This is just a preference but I call this 'data' or 'creatureData' to be a bit more explicit for other devs. The less ambiguous you make it the easier it is hop in code base and work. Again, just a thought, this works just fine as is.